libr/core/cmd_print.c

```diff
@@ -5886,7 +5886,7 @@ l = use_blocksize;
 							ut8 *data = malloc (len);
 							if (data) {
 								r_io_read_at (core->io, core->offset, data, len);
-								r_charset_encode_str (core->print->charset, out, out_len, data, len);
+								r_charset_encode_str (core->print->charset->db, out, out_len, data, len, false);
```

Comment:
nope
```suggestion
								r_charset_encode_str (core->print->charset, out, out_len, data, len, false);
```

Reply:
@trufae I planned to use r_charset_decode for both the ps (charset->db) and to w (charset->db_char_to_hex).